test/functional/test_framework/test_framework.py

```diff
@@ -189,6 +189,8 @@ def parse_args(self):
         parser.add_argument("--randomseed", type=int,
                             help="set a random seed for deterministically reproducing a previous test run")
         parser.add_argument("--timeout-factor", dest="timeout_factor", type=float, help="adjust test timeouts by a factor. Setting it to 0 disables all timeouts")
+        parser.add_argument("--v2transport", dest="v2transport", default=False, action="store_true",
```

Comment:
2aabbdf4de01ef1a3fb0c1bb86ef163c9fba587b Nice! Only three tests fail when I enable this for _all_ tests:
* p2p_node_network_limited.py
* p2p_timeouts.py
* rpc_net.py

Reply:
FWIW, I've started to adapt those tests for `--v2transport` support, and at least for p2p_node_network_limited.py and rpc_net.py there weren't too many changes needed (see https://github.com/theStack/bitcoin/commits/enable_v2_transport_for_more_tests).